### DISCUSSION TITLE
Remix IDE No injected provider found. Make sure your provider (e.g. MetaMask, ...) is active and running (when recently activated you may have to reload the page).

### DISCUSSION THREAD

**Discussion by ramziosta** [Cyfrin/foundry-full-course-cu#1264] [General]
I am trying to use the Remix IDE and when I try to use the Injected  Provider it doesn't accept the meta mask, I tried connecting and updating the Chrome extension on both Chrome and Firefox, as I saw on the community but it's not accepting it? Has anyone encountered this issue and how to solve it?

I did  the tips at this link:
https://ethereum.stackexchange.com/questions/83685/how-do-i-inject-web3-into-remix-ide
but didn't help me with my issue and could not find the settings in meta mask as the settings were different from 2 years ago per discussion

error: No injected provider found. Make sure your provider (e.g. MetaMask, ...) is active and running (when recently activated you may have to reload the page).

**Comment by xyz899** [comment]
## Hi,

### This issue can occur due to several reasons, mostly versions problems of MetaMask or Remix IDE. Here are some things you can try to troubleshoot the issue:

1. Check your browser version: Be sure you're using a recent version of your browser. Both MetaMask and Remix IDE work best with the latest versions of Chrome, Firefox, and Brave.
2. Update MetaMask  
3. Check for conflicting extensions: some browser extensions can interfere with MetaMask and Remix IDE. Try disabling other extensions, especially those related to privacy or security, and see if the problem persists.
4. Reset MetaMask: If none of the above steps work, try resetting MetaMask. Go to the MetaMask settings, click on 'Advanced', then 'Reset Account'. Be aware that this will delete all accounts stored in MetaMask. You'll have to re-add your accounts and networks afterward.

Last possibility is to Reinstall MetaMask. Hope you'll find the answer here

**Reply by ramziosta** [reply]
Thank you but non of those helped.

**Comment by tobezhanabi** [comment]
Do you have phantom wallet? if you do switch it off 
or try creating a new google chrome user

**Reply by ramziosta** [reply]
Thanks unfortunately that didn't help either

**Reply by TotalingArc** [reply]
Yes It was the Phantom! Pholana sux! get out before the house of cards crumbles

**Reply by EngrPips** [reply]
hmmmmmm🤔

**Reply by cesa87** [reply]
@tobezhanabi I love you man!

**Reply by EngrPips** [reply]
Awesome.

**Comment by kiwi6185** [comment]
Hi. I'm encountering the same issue. I've switched to using the MS Edge browser, and now my Metamask account appears there.

**Reply by EngrPips** [reply]
Interesting. Glad you unblocked yourself.

**Comment by s-di-cola** [comment]
had the same problem with brave: once I have linked metamask to the brave wallet it started working again.

**Reply by EngrPips** [reply]
Awesome, I am glad you figured it out.
